preload/preload_steps.go

```diff
@@ -0,0 +1,240 @@
+package preload
+
+import (
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/stepman/models"
+	"github.com/bitrise-io/stepman/stepman"
+)
+
+const (
+	workers       = 10
+	monthDuration = 24 * time.Hour * 31
```

Comment:
I'm pretty sure it's not a good idea 😄 

I think we should take the current wall clock time, subtract `PatchesSinceMonths`, then compare that datetime to the step publish datetime.

Reply:
It seems scary yes. Here is the comparison:
https://github.com/bitrise-io/stepman/blob/f66b89dccc9a8fc96684c81276fb818ba2365406/preload/preload_steps_util.go#L22
Went with this as wanted a simple way to parametrize and month seemed like a usable unit of time. I think could switch to days.
Could pass in a duration as well (then will have to do the conversion from days to duration at a level above).

Reply:
OK I now see that there is https://pkg.go.dev/time#Time.AddDate. Which in turn is timezone dependent :0.

Reply:
No, I'm okay with this month param, I just don't want to hardcode 1 month to N seconds because what we usually want is to go back N calendar months in the local timezone and at the current date.